public/js/smooth-scroll.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+$('.menu a').on('click', function(e) {
+  e.preventDefault();
+  var target = this.hash;
+  var $target = $(target);
+  var $targetOffset = $target.offset().top-80;
```

Comment:
For ease of reading, you typically want to add space around your operators: `$target.offset().top - 80`.

Also, why 80px? Is there something in the DOM that could be inspected to determine that height?